src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/NativeMemoryTests.cs

```diff
@@ -508,11 +508,11 @@ public void ZeroMemoryTest(uint size, uint offset)
         [InlineData(256, 143)]
         public void ZeroMemoryWithExactRangeTest(uint size, uint offset)
```

Comment:
Instead of all this casting, would it just make more sense to update the test signature?

Reply:
Yup, right. Done in 58e1e1ce03e4d822ffa965397caeeba3d42e074c 🙂